manifests/a/AVerMedia/RECentral/4.7.44.1/AVerMedia.RECentral.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.RECentral
+PackageVersion: 4.7.44.1
+PackageLocale: en-US
+Publisher: AVerMedia TECHNOLOGIES, Inc.
+PackageName: AVerMedia RECentral
+License: © 2022 AVerMedia Technologies, Inc.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate